IS/publiccode.yml

```diff
@@ -0,0 +1,118 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+applicationSuite: WSO2
+categories:
+  - it-security
+  - it-service-management
+  - identity-management
+description:
+  en:
+    documentation: 'https://wso2.com/identity-server/'
+    features:
+      - Easily add critical IAM features to your CX projects
+      - SSO, seamless login,forgotten password,account recovery and consent management
```

Comment:
Lets add 
- MFA (Multi-Factor Authentication)
- Passwordless Authenticatoion Push-Based Authentication and  Biometric Authentication
- Conditional Authentication for advanced CIAM usecases

Reply:
Updated accordingly